roles/openshift_facts/library/openshift_facts.py

```diff
@@ -623,7 +623,7 @@ def set_deployment_facts_if_unset(facts):
             service_type = 'atomic-openshift'
             if deployment_type == 'origin':
                 service_type = 'origin'
-            elif deployment_type in ['enterprise', 'online']:
+            elif deployment_type in ['enterprise']:
```

Comment:
This change will affect managing 3.0 based online envs.

My suggestion would be to either tag the repo prior to merging and use that tag for all actions on 3.0 deployments, and use the appropraite branch for 3.1 upgrades and 3.1 envs.

Either that or you will want to add a new deployment_type for online 3.1 deployments, but that would cause additional issues with having to re-tag ec2 instances, etc.